ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/main.cpp

```diff
@@ -30,6 +30,9 @@
 
 int main(int argc, char *argv[])
 {
+  // Enable OpenGL
```

Comment:
This should be changed throughout. Hopefully it is a trivial find and replace.
```suggestion
  // Enforce OpenGL
```

So it's clearer, we should make the comment state that we are enforcing/requiring OpenGL for now. "Enable" could be confusing as to what we're doing.

Reply:
gotcha that makes sense, just changed it!